### PR TITLE
New: Nested Tabs - Refactor show/hide tab content callbacks to be more generic [ED-9454]

### PR DESCRIPTION
**Pull request by udidol**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* New: Mega Menu - Add Content Width Control to the Layout section (Full width)

### CODE REVIEW

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 1]
```diff
@@ -145,10 +145,12 @@ export default class NestedTabs extends Base {
 			'aria-expanded': 'false',
 		} );
 
-		$activeContent[ settings.hideTabFn ]();
+		$activeContent[ settings.hideTabFn ]( 0, () => this.onHideTabContent( $activeContent ) );
 		$activeContent.attr( 'hidden', 'hidden' );
 	}
 
+	onHideTabContent() {}
```
Comment (louiswol94) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:152]: @udidol Should you not rather add the param here so the signature would be the same?

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 2]
```diff
@@ -175,14 +177,16 @@ export default class NestedTabs extends Base {
 
 		$requestedContent[ settings.showTabFn ](
 			animationDuration,
-			() => {
-				elementorFrontend.elements.$window.trigger( 'elementor-pro/motion-fx/recalc' );
-				elementorFrontend.elements.$window.trigger( 'elementor/nested-tabs/activate', $requestedContent );
-			},
+			() => this.onShowTabContent( $requestedContent )
```
Comment (louiswol94) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:180]: @udidol Missing trailing comma

**Comment by louiswol94** [comment]
@udidol Should we rename this PR title? It does not seem like it has anything specific to do with the "Content Width Control"

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js (hunk 2 above)
Comment (udidol) [reply modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:180]: There is no trailing comma in function calls, only in arrays

**Review by louiswol94 — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**